src/drivers/Qt/ConsoleVideoConf.cpp

```diff
@@ -736,7 +736,7 @@ void ConsoleVideoConfDialog_t::openGL_linearFilterChanged( int value )
    }
 }
 //----------------------------------------------------
-#ifdef WIN32
+#if defined(WIN32) && (QT_VERSION_MAJOR < 6)
 void ConsoleVideoConfDialog_t::winFullScreenBorderChanged(int value)
```

Comment:
This particular function should have built fine on Qt 6. I don't think this is necessary

Reply:
It didn't work with Qt-6.6.2.

Reply:
The winFullScreenBorderChanged method prototype definition in the header needs to have the same processor guard.